index.html

```diff
@@ -165,11 +165,13 @@ <h1><img src="assets/infragram-logo.png" alt="Infragram logo image." style="floa
             </div>
           </div>
 
-          <div class="zone last" id="save-zone" style="display:none;">
```

Comment:
Ah, i see! Thank you @stylebytemi however one reason we have the buttons hidden initially is because before you begin uploading something, there is nothing to save, so we thought it best to hide them. An alternative could be to have them present but greyed out, but it still adds some perhaps unnecessary clutter to the page before those buttons can even be used. What do you think?

Reply:
Hello @jywarren, I assume a design proposal to solve the unnecessary clutter has been raised multiple times. However, I will suggest we leave it displayed to avoid confusion for users. My design suggestion would also be that of others which is to use a bootstrap property such as dropdown for clearer display. we can also use react routing and give each column(Div)  used in toolbar a new tag which directly links to a clearer and well detailed page. I'm sorry for the late reply. Thanks for your time.